src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
   // write code here
-}
+  const inverse = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(inverse, robot[key])) {
```

Comment:
Instead of using `Object.prototype.hasOwnProperty.call()`, you can use the `in` operator to check if a property exists in an object. It's simpler and more readable. For example: `if (robot[key] in inverse)`.